test/AutoinstSoftware_test.rb

```diff
@@ -34,7 +33,31 @@
 
       expect(Yast::AutoinstData.send(:post_packages)).to eq(["a1","a2"])
     end
-
   end
 
+  describe "#Export" do
+    it "puts product definition into the exported profile" do
+      expect(Yast::Product)
```

Comment:
np: I would use `allow` as you are setting pre-conditions. But I do not have a strong opinion about it.